src/apps/app_election.erl

```diff
@@ -4,11 +4,11 @@
 -include("ar.hrl").
 -include_lib("eunit/include/eunit.hrl").
 
-%%% The Archain OpenElections monitor node.
+%%% The Arweave OpenElections monitor node.
 %%% Listens for votes being cast in elections. Returns results when asked.
 
 %% How should the app identify itself in the 'type' field?
--define(APP_TYPE_ID, "Archain.OpenElection").
+-define(APP_TYPE_ID, "Arweave.OpenElection").
```

Comment:
@samcamwilliams I am not sure about this one, what would be the implications of this?